addons/packages/contour/1.17.1/package.yaml

```diff
@@ -71,7 +71,6 @@ spec:
                   default: Local
                 annotations:
                   type: object
-                  additionalProperties: string
```

Comment:
For Contour, can we switch these to:

```yaml
additionalProperties:
  type: string
```

Reply:
I wonder if this is a better fix overall?

Reply:
I converted all `additionalProperties: string` references to

```
additionalProperties:
  type: string
```